cognite_toolkit/_cdf_tk/prototypes/interactive_init.py

```diff
@@ -31,43 +34,26 @@
 )
 
 
-def get_packages() -> dict[str, dict[str, Any]]:
-    return {
-        "quickstart": {
-            "title": "Quick Start: A set of modules for a CDF quick start project.",
-            "items": {
-                "sap_data_pipeline": {
-                    "title": "SAP Data Pipeline",
-                },
-                "pi_data_pipeline": {
-                    "title": "PI Data Pipeline",
-                },
-                "mqtt_data_pipeline": {
-                    "title": "MQTT Data Pipeline",
-                },
-                "files_contextualization": {
-                    "title": "Files Contextualization",
-                },
-                "asset_data_transformation": {
-                    "title": "Asset Data Transformation",
-                },
-                "infield": {
-                    "title": "Infield",
-                },
-            },
-        },
-        "examples": {
-            "title": "Examples: a set of example modules for inspiration",
-            "items": {
-                "cdf_data_pipeline_asset_valhall": {"items": {}},
-                "cdf_data_pipeline_files_valhall": {"items": {}},
-            },
-        },
-        "reference": {
-            "title": "All supported resources as reference",
-            "items": {"workflow": {}, "transformations": {}, "functions": {}, "groups": {}},
-        },
-    }
+class Packages:
+    def __init__(self) -> None:
+        self._packages: dict[str, dict[str, Any]] | None = None
+
+    @property
+    def content(self) -> dict[str, dict[str, Any]] | None:
+        if self._packages is None:
+            self._packages = self._get_packages()
+        return self._packages
+
+    def _get_packages(self) -> dict[str, dict[str, Any]] | None:
+        packages = {}
+
+        for module in _packages.__all__:
+            manifest = Path(_packages.__file__).parent / module / "manifest.yaml"
+            if not Path(manifest).exists():
+                raise FileNotFoundError(f"Manifest file not found for package {module}")
+            with open(manifest) as file:
+                packages[manifest.parent.name] = yaml.CSafeLoader(file.read()).get_data()
+        return packages
```

Comment:
I think this is simpler and more robust. If the manifest is missing it will lead to a bad user experience, instead a warning is appropriate.
```suggestion
class Packages(dict, MutableMapping[str, dict[str, Any]]):
    @classmethod
    def load(cls) -> Packages:
        packages = {}
        for module in _packages.__all__:
            manifest = Path(_packages.__file__).parent / module / "manifest.yaml"
            if not manifest.exists():
                warnings.warn(f"Bug in Cognite-Toolkit. Manifest file not found for package {module}")
                continue
            content = manifest.read_text()
            if yaml.__with_libyaml__:
                packages[manifest.parent.name] = yaml.CSafeLoader(content).get_data()
            else:
                packages[manifest.parent.name] = yaml.SafeLoader(content).get_data()
        return cls(packages)

    @property
    def available(self) -> list[str]:
        return sorted(self.keys())
```